_posts/2016-04-22-elasticsearch-module_2.md

```diff
@@ -193,21 +180,14 @@ contacts 3     r         STARTED    0   79b 127.0.1.1 Solano López
 
 (*)prirep: primary/replica
```

Comment:
Ojo que este `*` puede hacer que el resto del texto aparezca en itálica de ahí en más. Es preferible simplemente aclarar algo como

> Nota: en el extracto anterior, `prirep` hace referencia a `primary/replica`, siendo identificados con una `p` los nodos primarios y con una `r` sus réplicas.